experimental/packages/opentelemetry-browser-detector/src/types.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export type UserAgentData = {
+  brands: string[],
```

Comment:
The type should be an array of `NavigatorUABrandVersion` objects according to https://wicg.github.io/ua-client-hints/#dom-navigatoruadata-brands.

Reply:
fixed the type, couldn't import the type as it is (experimental).